### ISSUE TITLE
VoiceOver on IOS does not find links in list views

### ISSUE THREAD

**Issue by smukkejohan** [Notalib/LYT#117]
I found the reason for this problem. jQuery mobile inserts `aria-hidden="true"` on `.ui-btn-inner`element in listviews when a link element is injected. Why? - I have no fucking clue, `aria-hidden="true"` is an WAI-ARIA property that tells screen readers and other assistive devices that the element should be considered hidden from the user (http://www.w3.org/TR/wai-aria/states_and_properties#aria-hidden).
